test/utils.js

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright 2020 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+/* eslint-disable no-undef */
+/* eslint-disable camelcase */
+const { fetch } = require('@adobe/helix-fetch');
+const assert = require('assert');
+
+const testDomain = process.env.TEST_DOMAIN;
+
+/**
+ * gets a list of urls from by sending a request to helix-run-query
+ *
+ * return list of objects i.e. {base: 'blog.adobe.com', branch: 'blog.adobe.hlx.page'}
+ */
+async function getUrls() {
```

Comment:
```suggestion
async function getMostVisitedUrls() {
```

Nitpicking, I just find `getUrls` too generic compared to what it really does.